docs/benchmarking-workflow-corefx.md

```diff
@@ -0,0 +1,253 @@
+
+The next step is to send a PR to this repository with the aforementioned benchmark. Our automation is going to run this benchmark and export the results to our reporting system. When your fix to CoreFX gets merged, our reports are going to show the difference. It also helps us to keep track of the old performance bugs and make sure that they never come back.
+
+### Profiling
+
+The real performance investigation starts with profiling. To profile the benchmarked code and produce an ETW Trace file ([read more](./benchmarkdotnet.md#Profiling)):
+
+```
+dotnet run -f netcoreapp3.0 --profiler ETW --filter $YourFilter 
+```
+
+The benchmarking tool is going to print the path to the `.etl` trace file. You should open it with PerfView or Windows Performance Analyzer and start the analysis from there. If you are not familiar with PerfView, you should watch [PerfView Tutorial](https://channel9.msdn.com/Series/PerfView-Tutorial) by Vance Morrison first. It's an investment that is going to pay off very quickly.
+
+```log
+// * Diagnostic Output - EtwProfiler *
+Exported 1 trace file(s). Example:
+C:\Projects\performance\artifacts\20190215-0303-51368\Benchstone\BenchF\Adams\Test.etl
+```
+
+If profiling using the `--profiler ETW` is not enough, you should use a different profiler. When attaching to a process please keep in mind that what you run in the console is Host process, while the actual benchmarking is performed in dedicated processes. If you want to disable this behavior, you should use [InProcessToolchain](./benchmarkdotnet.md#Running-In-Process).
+
+### Running against Older Versions
+
+BenchmarkDotNet has some extra features that might be useful when doing performance investigation:
+
+- You can run the benchmarks against [multiple Runtimes](./benchmarkdotnet.md#Multiple-Runtimes). It can be very useful when the regression has been introduced between .NET Core releases, for example: between netcoreapp2.2 and netcoreapp3.0.
+- You can run the benchmarks using provided [dotnet cli](./benchmarkdotnet.md#dotnet-cli). You can download few dotnet SDKs, unzip them and just run the benchmarks to spot the version that has introduced the regression to narrow down your investigation.
+- You can run the benchmarks using few [CoreRuns](./benchmarkdotnet.md#CoreRun). You can build the latest CoreFX in Release, create a copy of the folder with CoreRun and use git to checkout older commit. Then rebuild the CoreFX and run the benchmarks against the old and new builds. This can narrow down your investigation to the commit that has introduced the bug.
+
+### Confirmation
+
+When you identify and fix the regression, you should use [ResultsComparer](../src/tools/ResultsComparer/README.md) to confirm that you have solved the problem. Please remember that if the regression was found in a very common type like `Span<T>` and you are not sure which benchmarks to run, you can run all of them using `--filter *`.
+
+## Local CoreCLR Build
```

Comment:
@stephentoub @ahsonkhan you both asked for that, I have tested that and made sure that it works as expected

Reply:
Double confirmation I did it last week https://github.com/dotnet/coreclr/pull/22599

Reply:
@MarcoRossignoli thanks for the confirmation!